packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -382,7 +390,7 @@ Future<Null> _buildGradleProjectV2(
   status.stop();
 
   if (exitCode != 0)
-    throwToolExit('Gradle build failed: $exitCode', exitCode: exitCode);
+    throwToolExit('Gradle task $assembleTask failed with exit code $exitCode', exitCode: exitCode);
```

Comment:
how do we get into "verbose mode" with Gradle?  If I ever ran into a compilation issue I would definitely want the task name.  I find it notoriously difficult to even figure out where to start debugging most Gradle failures...

Reply:
Yeah, I added this because otherwise it's really hard to know what gradle is even trying to currently do. At least this way you know what gradle task to start with.

We definitely need better handling of Gradle output in general, but if something is wrong with the build, and it's not a compiler error (which this PR is trying to improve), then you need gradle information to figure it out.